src/main/java/io/debezium/connector/spanner/config/BaseSpannerConnectorConfig.java

```diff
@@ -181,7 +181,7 @@ public abstract class BaseSpannerConnectorConfig extends CommonConnectorConfig {
             .withType(Type.STRING)
             .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR, 1))
             .withWidth(Width.MEDIUM)
-            .withImportance(Importance.HIGH)
+            .withImportance(Importance.LOW)
```

Comment:
IMHO if reduction of importance is done then it should be `MEDIUM`. Same for the one below.

Reply:
Why would reduction in importance be needed here?

Reply:
These JSON key options are not required when using workload identity authentication. The importance should be changed.